importer/process_import.py

```diff
@@ -518,6 +521,21 @@ def validate_extent(self, extent_rect):
         else:
             return True
 
+    @staticmethod
+    def _create_field_and_value(attribute_name, attribute_value_as_str):
+        if attribute_name == GPX_FIELDS["time"]:
+            dt = QDateTime.fromString(attribute_value_as_str, Qt.ISODate)
+            field_value = dt if dt.isValid() else attribute_value_as_str
+            field_type = QVariant.DateTime
+        elif attribute_name == GPX_FIELDS["ele"]:
+            field_value = float(
```

Comment:
If it is too long, it is better and more readable to use multi-line statement according to PEP 8.  So it is better to use multi line, if `attribute_value_as_str is not None`. Also it is good to have the same condition in the first case for time.